CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ for further information about branching and tagging conventions.
 ## [Unreleased]
 #### Fixed
 - A bug with localization
+- Error with reference length
```

Comment:
Remove this msg
Add section 'Changed'
Msg: 'REFERENCE_MAX_LENGTH from 255 to 64'